public/leaderboardList.php

```diff
@@ -435,11 +431,12 @@ function UpdateLeaderboard(lbID) {
         $prevOffset = $offset - $maxCount;
         echo "<a href='/achievementList.php?s=$sortBy&amp;o=$prevOffset'>&lt; Previous $maxCount</a> - ";
     }
-    if ($listCount == $maxCount) {
-        // Max number fetched, i.e. there are more. Can goto next 25.
-        $nextOffset = $offset + $maxCount;
-        echo "<a href='/achievementList.php?s=$sortBy&amp;o=$nextOffset'>Next $maxCount &gt;</a>";
-    }
+    // TODO fix pagination - $listCount is not being increased
+    // if ($listCount == $maxCount) {
+    //     // Max number fetched, i.e. there are more. Can goto next 25.
+    //     $nextOffset = $offset + $maxCount;
+    //     echo "<a href='/achievementList.php?s=$sortBy&amp;o=$nextOffset'>Next $maxCount &gt;</a>";
+    // }
```

Comment:
TODO needs to be addressed

Reply:
Doesn't look like this worked before.
https://retroachievements.org/leaderboardList.php?g=3680
It incorrectly links to `achievementList.php`, the listCount is not calculated properly either. Nice to see PHPStan finding bugs like this :)
I'll fix that.

Reply:
Addressed in https://github.com/RetroAchievements/RAWeb/pull/1404